src/index.js

```diff
@@ -24,6 +24,10 @@ app.use(
   })
 );
 
+//serving resources
+const path = require('path');
```

Comment:
Maybe put this line on line 5 instead since it's an import from an external library, just so we have consistency.